lib/ansible/cli/galaxy.py

```diff
@@ -1148,15 +1136,11 @@ def execute_list_role(self):
                     _display_role(gr)
                     break
                 warnings.append("- the role %s was not found" % role_name)
+            elif role_name and not os.path.exists(role_path):
```

Comment:
you  dont need role_name since to get here it is already false you want it to be false anyways